Dockerfile-8.1

```diff
@@ -0,0 +1,147 @@
+FROM behance/docker-nginx:10.2.0 as output
+LABEL maintainers="Behance Team <dev-behance@adobe.com>"
+
+# Set TERM to suppress warning messages.
+ENV CONF_PHPFPM=/etc/php/8.1/fpm/php-fpm.conf \
+    CONF_PHPMODS=/etc/php/8.1/mods-available \
+    CONF_FPMPOOL=/etc/php/8.1/fpm/pool.d/www.conf \
+    CONF_FPMOVERRIDES=/etc/php/8.1/fpm/conf.d/overrides.user.ini \
+    APP_ROOT=/app \
+    SERVER_WORKER_CONNECTIONS=3072 \
+    SERVER_CLIENT_BODY_BUFFER_SIZE=128k \
+    SERVER_CLIENT_HEADER_BUFFER_SIZE=1k \
+    SERVER_CLIENT_BODY_BUFFER_SIZE=128k \
+    SERVER_LARGE_CLIENT_HEADER_BUFFERS="4 256k" \
+    PHP_FPM_MAX_CHILDREN=4096 \
+    PHP_FPM_START_SERVERS=20 \
+    PHP_FPM_MAX_REQUESTS=1024 \
+    PHP_FPM_MIN_SPARE_SERVERS=5 \
+    PHP_FPM_MAX_SPARE_SERVERS=128 \
+    PHP_FPM_MEMORY_LIMIT=256M \
+    PHP_FPM_MAX_EXECUTION_TIME=60 \
+    PHP_FPM_UPLOAD_MAX_FILESIZE=1M \
+    PHP_OPCACHE_MEMORY_CONSUMPTION=128 \
+    PHP_OPCACHE_INTERNED_STRINGS_BUFFER=16 \
+    PHP_OPCACHE_MAX_WASTED_PERCENTAGE=5 \
+    PHP_OPCACHE_ENABLE_CLI=1 \
+    PHP_ENGINE_VERSION=20200930 \
+    CFG_APP_DEBUG=1
+
+COPY ./scripts/ /scripts/
+
+# - Update security packages
+RUN /bin/bash -e /security_updates.sh
+
+# - Add PHP and support packages
+RUN apt-get install -yqq -f --no-install-recommends \
```

Comment:
Can you combine this w/ the previous RUN command, similar to https://github.com/behance/docker-php/blob/master/Dockerfile-8.0#L38-L39 ?

Reply:
Updated.